### PR TITLE
[PI-119] FHIR/Swagger/Pydantic

### PR DESCRIPTION
**Pull request by jaklinger**

### CODE REVIEW

File: .github/actions/components/caching/action.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ runs:
           **/tfplan
           **/swagger.yaml
           **/infrastructure/terraform/**/output.json
+          .downloads
```
Comment (Rohoolio) [inline .github/actions/components/caching/action.yml:25]: This relates to all the downloady bits, im sure we didnt do it this way with NRLF so wondering whats different this time for pulling down the JAR file? The only thing I can think of is that we are doing this on the fly in the CI?

File: .tool-versions [hunk 2]
```diff
@@ -4,3 +4,4 @@ python 3.11.4
 terraform 1.5.7
 jq 1.7
 yq 4.34.2
+nodejs 20.9.0
```
Comment (Rohoolio) [inline .tool-versions:7]: Is this now needed on the packer image? Starting to sound like that ticket could do with being brought in soon haha

File: .github/actions/components/caching/action.yml (hunk 1 above)
Comment (jaklinger) [reply .github/actions/components/caching/action.yml:25]: i just needed somewhere to download the JAR file to :D don't mind where it goes, but somewhere at the repo root that is gitignored. What do you suggest?

File: .tool-versions (hunk 2 above)
Comment (jaklinger) [reply .tool-versions:7]: ah yeah i'll add that to the ticket

**Pull request merged**